python/ray/rllib/models/multiagentfcnet.py

```diff
@@ -42,7 +42,11 @@ def _init(self, inputs, num_outputs, options):
         shared_model = shared_model or isinstance(shared_model, list)  # for heterogeneous shared models
         # the list indicates how many agents should share each model i.e.
         # list [k1, k2, ...] indicates that first k1 agents share a model, then k2 share a model, etc.
-        shared_model_list = custom_options.get("multiagent_shared_model", [len(hiddens)])
+        # Users can pass either True/False or a list
```

Comment:
It seems that if the use passes True/False, it will still evaluate as not a list, and thus fall to the else statement (regardless of if the user indicates true or false). What is the expected behavior?

Reply:
Ah, I see the comment is confusing. Basically, I want users to be able to pass True/False or a list, and have the behavior be
True: All agents share a model
False: No agents share a model
List=[k1,k2,etc.]: First k1 agents share a model, next k2 agents share a model. I think this is implemented correctly above?

Reply:
The way I'm reading it, regardless of True or False, the following will be executed. What indicates whether `len(hiddens)` is all shared or not?
```
shared_model_list = [len(hiddens)]
```

Reply:
So if the shared model is a bool instead of a list , it defaults to setting all the models to be the same.